### DISCUSSION TITLE
Migrate to esbuild and tailwind?

### DISCUSSION THREAD

**Discussion by dorianmariecom** [rubygems/rubygems.org#2866] [Ideas]
would be nicer, what do you think?

**Comment by sonalkr132** [comment]
note that we aren't using webpack https://github.com/rubygems/rubygems.org/issues/2126. clipboard and github-buttons are our only third party js dependencies. it would be nice to keep them updated with build pipeline but manual has worked for us so far. github-buttons is also heavily modified.

**Comment by simi** [comment]
Yeah, IMHO currently there is no benefit actually. It will make things even more complicated on deployment side AFAIK.
